draft-ietf-core-dns-over-coap.md

```diff
@@ -211,8 +211,8 @@ The term "constrained nodes" is used as defined in {{-constr-nodes}}.
 The terms "payload" and "body" are used as defined in {{-coap-blockwise, Section 2}}.
 Note that, when block-wise transfer is not used, the terms "payload" and "body" are to be understood as equal.
 
-For better readability, examples in this document the binary payload is provided in hexadecimal representation as well as a human-readable format.
-In the actual message, however, it would be encoded in the binary message format defined in {{-dns}}.
+For better readability, examples in this document the binary payload and resource records are provided in hexadecimal representation as well as a human-readable format.
```

Comment:
this sentence appears still broken. it should be "examples in this document of the binary payload and resource records are provided [...]"